src/pages/Dashboard/EarnDashboard.tsx

```diff
@@ -1,44 +1,28 @@
-import { ArrowForwardIcon } from '@chakra-ui/icons'
-import { Button, Flex, Heading } from '@chakra-ui/react'
+import { Flex } from '@chakra-ui/react'
 import { memo } from 'react'
 import { useTranslate } from 'react-polyglot'
-import { Link as NavLink } from 'react-router-dom'
 
 import { SEO } from '@/components/Layout/Seo'
 import { DeFiEarn } from '@/components/StakingVaults/DeFiEarn'
-import { RawText } from '@/components/Text'
 
 const alignItems = { base: 'flex-start', md: 'center' }
 const padding = { base: 4, xl: 0 }
-const arrowForwardIcon = <ArrowForwardIcon />
 
 const EarnHeader = () => {
   const translate = useTranslate()
 
   return (
     <Flex alignItems={alignItems} px={padding} flexWrap='wrap'>
       <SEO title={translate('navBar.defi')} />
-      <Flex width='full' justifyContent='space-between' alignItems='center'>
-        <Heading fontSize='xl'>{translate('defi.myPositions')}</Heading>
-        <Button
-          colorScheme='purple'
-          variant='ghost'
-          as={NavLink}
-          to='/earn'
-          size='sm'
-          ml='auto'
-          rightIcon={arrowForwardIcon}
-        >
-          {translate('defi.viewAllPositions')}
-        </Button>
```

Comment:
blocking: this should go away, but we should keep the heading and subheading

Reply:
Done in [8db94c1](https://github.com/shapeshift/web/pull/9086/commits/8db94c1ebd64971e4a6f3dd422478112132f863f)